Source: Hong Hao <oahong@gmail.com>
Upstream: No
Reason: Introduce zhengma support to fcitx, also remove loads of crap from system
    (those code table sources/xpm icons are not needed)
--- fcitx-scm/xpm/Makefile.am.orig	2009-05-16 00:22:38.933554435 +0800
+++ fcitx-scm/xpm/Makefile.am	2009-05-16 00:27:55.099577940 +0800
@@ -1,5 +1,4 @@
-xpmdir = $(pkgdatadir)/xpm
-xpm_DATA = about_icon.xpm chnPunc.xpm engPunc.xpm ft-0.xpm ft-1.xpm \
+EXTRA_DIST = about_icon.xpm chnPunc.xpm engPunc.xpm ft-0.xpm ft-1.xpm \
 	fullcorner.xpm gbk-0.xpm gbk-1.xpm halfcorner.xpm lock-0.xpm \
-	lock-1.xpm logo.xpm lx-0.xpm lx-1.xpm next.xpm prev.xpm vk-1.xpm vk.xpm
-EXTRA_DIST = $(xpm_DATA)
+	lock-1.xpm logo.xpm lx-0.xpm lx-1.xpm next.xpm prev.xpm  \
+	vk-1.xpm vk.xpm inactive.xpm active.xpm
--- fcitx-scm.orig/data/tables.conf	2009-05-03 22:42:00.000000000 +0800
+++ fcitx-scm/data/tables.conf	2009-05-15 23:42:41.630146450 +0800
@@ -19,6 +19,19 @@
 符号=zzzz
 
 [码表]
+名称=郑码
+码表=zhengma.mb
+调频=2
+拼音=1
+拼音键=`
+自动上屏=-1
+空码自动上屏=-1
+自动词组=1
+保存自动词组=3
+精确匹配=0
+提示编码=0
+
+[码表]
 名称=五笔拼音
 码表=wbpy.mb
 调频=2
--- fcitx-scm.orig/data/Makefile.am	2009-03-03 21:48:59.000000000 +0800
+++ fcitx-scm/data/Makefile.am	2009-05-15 23:40:58.144154123 +0800
@@ -1,10 +1,10 @@
 tabledir = $(pkgdatadir)/data
 table_DATA = cj.mb erbi.mb punc.mb pybase.mb pyphrase.mb pySym.mb sp.dat \
-	     wbx.mb wanfeng.mb wbpy.mb qxm.mb zrm.mb db.mb \
-	     tables.conf vk.conf gbks2t.tab config AutoEng.dat \
-             pyPhrase.org qxm.txt zrm.txt erbi.txt wbpy.txt gbkpy.org db.txt \
-             wanfeng.txt cj.txt wbx.txt
-EXTRA_DIST = $(table_DATA)
+	     wbx.mb wanfeng.mb wbpy.mb qxm.mb zrm.mb zhengma.mb db.mb \
+	     tables.conf vk.conf gbks2t.tab config AutoEng.dat
+table_SOURCE = pyPhrase.org qxm.txt zrm.txt erbi.txt wbpy.txt gbkpy.org db.txt \
+             wanfeng.txt cj.txt wbx.txt zhengma.txt
+EXTRA_DIST = $(table_DATA) $(table_SOURCE)
 
 mb_data = cj.mb erbi.mb wanfeng.mb wbpy.mb qxm.mb db.mb wbx.mb zrm.mb
 
